src/js/edu-benefits/definitions/educationProgram.js

```diff
@@ -2,31 +2,18 @@ import _ from 'lodash/fp';
 
 import { showSchoolAddress } from '../utils/helpers';
 import * as address from '../../common/schemaform/definitions/address';
-import educationTypeUISchema from '../definitions/educationType';
+import educationTypeUISchema from './educationType';
 
 export const uiSchema = {
   'ui:order': ['name', 'educationType', 'address'],
-  address: address.uiSchema(),
+  address: _.merge(address.uiSchema(), {
```

Comment:
This is branched from https://github.com/department-of-veterans-affairs/vets-website/pull/5157 so give credit to @cvalarida, but yes, looks like we now have the `showSchoolAddress()` helper so don't need to write a custom callback.

@cvalarida I'm not sure what issue this connects to  (only ask for address for certain education types)--can you add to the PR description if there is one? And are we sure it applies to both the 5490 and 1990e? I would assume/hope so, but you never know.